### DISCUSSION TITLE
How to translate placeholder text (in TextInput form element)?

### DISCUSSION THREAD

**Discussion by code-reflex** [filamentphp/filament#6764] [Help] [answered]
Newbie here trying Filament. I am setting a placeholder text in TextInput component. E.g. "Enter your email here".  How to translate it to, say, Spanish using the translation options available?

**Comment by danharrin** [comment, marked as answer]
`->placeholder(__('translation string here'))`, we use Laravel's translation system

**Reply by code-reflex** [reply]
Thank you. Why did I not think of that? :(
